src/release.js

```diff
@@ -470,5 +471,66 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
 
             log.done(`${data.extension.name} is clean`);
         },
+
+        /**
+         * Select releasing branch
+         * - picking version-to-release CLI option and find branch with version on it
+         * - or find the biggest version and find branch with version on it
+         */
+        async selectReleasingBranch() {
+            // Filter all branches to the ones that have release in the name
+            await gitClient.fetch({'--prune': true});
+            const allBranches = await gitClient.getLocalBranches();
+
+            if (versionToRelease) {
+                const branchName = `remotes/${origin}/${branchPrefix}-${versionToRelease}`;
+                if (allBranches.includes(branchName)) {
+                    data.releasingBranch = branchName;
+                    data.version = versionToRelease;
+                }
+            } else {
+                const branchName = `remotes/${origin}/${branchPrefix}`;
```

Comment:
Technically it's a `partialBranchName` at this stage

Reply:
yes, ok.